draft-ietf-taps-interface.md

```diff
@@ -1045,26 +1045,53 @@ Type:
 : Enumeration
 
 Default:
-: Disabled
+: Disabled for connections created through initiate and rendezvous, Passive for listeners
 
-This property specifies whether an application wants to take advantage of
+This property specifies whether and how applications want to take advantage of
 transferring data across multiple paths between the same end hosts. Using
-multiple paths allows connections to migrate between interfaces as
-availability and performance properties change. Possible values are:
+multiple paths allows connections to migrate between interfaces or aggregate bandwidth
+as availability and performance properties change. Possible values are:
 
 Disabled:
-: The connection will not attempt using multiple paths once established
+: The connection will not allow using multiple paths once established
+
+Passive:
+: The connection will not actively try using multiple paths once established, but allow the peer endpoint to initiate using multiple paths and migrate existing connections to other paths.
 
 Handover:
-: The connection should attempt to migrate between different paths upon interface availability changes
+: The connection should only attempt to migrate between different paths when the original path is lost or becomes unreliable.
+The actual thresholds to declare a path unreliable are implementation specific.
 
 Interactive:
-: The connection should attempt to use multiple paths in response to loss or delay upon individual paths
+: The connection should attempt to use multiple paths in parallel in order to minimize loss and delay. The actual strategy is implementation specific and  my depend on the multipath protocol used, but should not aim to exceed the bandwidth provided by the best of the available paths.
+
+Passive-Interactive:
+: As Interactive, but only if using multiple paths was initiated by the peer endpoint.
```

Comment:
I agree with @gorryfair and @abrunstrom who call this "fuzzy" in #437 (this applies to both "Interactive" and "Passive-Interactive". Do we need this now, given that we don't seem to have any RFC explaining how to best implement this?  I think it's cool functionality, but I question whether this should be inside a Transport System - would an application be able to trust this to implement just the right trade-off for it, between e.g. capacity usage and delay minimization?

Side note: if we do keep it, I think the text of this update is better.

Reply:
This is a slightly extended version of Apple's implementation and can be easily implemented, e.g., for MPTCP by choosing the right path manager. It is a matter of choosing "Just use the lowest RTT path" vs. "use all paths"

Reply:
So if you have a long-RTT path with potentially a lot of extra capacity as well as a short-RTT path, this is always going to pick the short-RTT one?  Even if the capacity on the short-RTT one is quite tiny and the RTT difference isn't very large? Or else, how do you choose these thresholds?  => yes there CAN be a simple implementation, but there are also trade-offs in the design.

Reply:
I assume this is a weird corner case, but yes there are trade-offs the implementation must make. Do we need text on that?

Reply:
I don't think it's a weird corner case; I just don't think that many applications would want to say "give me the lower latency path, at all cost, no matter how miniscule the difference, no matter how much capacity I lose". I'm not proposing to address this with some discussion in text - I'm proposing not to delegate the algorithm that chooses paths on this basis to the transport system, but leaving it inside the application instead.

Reply:
Leaving it to the application may seem attractive, but is not really feasible in practise as it quickly gets very complicated. For instance, if you ask for a packet to be sent on one path and this packet is lost, are you allowed to retransmit it on another path for reliability? What if the path is lost before you manged to send the packet at all?
I think the updated text is an improvement.

Reply:
Ok, glad to hear that you like this - I don't personally have a strong opinion, and I agree that it quickly gets complicated!  (but that's also why I felt quite insecure about simply delegating this to the transport system). I would like to hear @tfpauly's opinion too, though.

Reply:
I think having "passive" here mixes up two things, one is the question of who will establish paths and the other one is how to use paths. These are separate points and should not be mixed together.